### DISCUSSION TITLE
GitHub Android Crash

### DISCUSSION THREAD

**Discussion by nandgator** [community/community#33605] [Mobile] [answered]
Goto https://github.com/oven-sh/bun in the app. Wait a few seconds. Try scrolling. App restarts.

**Comment by stevepopovich** [comment]
Hey Joe! Do you mind sharing your device and OS version? Does this happen consistently?

We have seen this happen occasionally with these large readmes that exist on repos.

**Comment by nandgator** [comment]
> Hey Joe! Do you mind sharing your device and OS version? Does this happen consistently?

Yes, at least for that repository.

OS: Android 12 (MIUI 13)
Device: Redmi Note 10T 5G

> We have seen this happen occasionally with these large readmes that exist on repos.

Mm.. figured

**Reply by stevepopovich** [reply]
This has been a long-standing issue for us. We will look into improvements to hopefully fix this. Thanks!

**Comment by nandgator** [comment, marked as answer]
Fix present in `GitHub Mobile v1.129.0-beta (10148)`
